themes/default_symbols.fish

```diff
@@ -36,6 +36,9 @@ set FLSYM_VFISH             "\u2635"
 # Symbol for CONDA segment
 set FLSYM_CONDA             "\U223F"
 
+# Symbol for SCREEN segment
+set FLSYM_SCREEN            "\U239A"
+
```

Comment:
Can we use the lowercase `\u` as a prefix to stay consistent with the other symbols as they are 4 digit UTF-8 escape sentence (`\uXXX` vs `\UXXXXXXXX` which is 8 digit escape).

PS: did not notice for the FLSYM_CONDA that it was also in the `\U` form.